encode/httpx
# Apply PEP-621

This was brought on https://github.com/encode/starlette/pull/1359.

setuptools released 0.61.0 yesterday, and brought to light the PEP-621 which makes this PR possible. Most of the PR was inspired on this page: https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html

Notes:
- ~I'll continue later on. I need to check if everything is included on the metadata files.~

## How to test?

- `./scripts/build` to see that wheel and `tar.gz` are generated
- `python -m pip install -e .` for editable installs
